trustd/src/main.rs

```diff
@@ -84,21 +85,23 @@ impl Trustd {
             let current_dir = env::current_dir()?;
             let work_dir = current_dir.join(".trustify");
             let db_dir = work_dir.join("postgres");
+            let data_dir = work_dir.join("data");
+            create_dir_all(&data_dir)?;
             let settings = Settings {
                 username: self.database.username.clone(),
                 password: self.database.password.clone(),
                 temporary: false,
                 installation_dir: db_dir.clone(),
+                data_dir,
                 ..Default::default()
             };
 
             let mut postgresql = PostgreSQL::new(PostgreSQL::default_version(), settings);
-            postgresql.setup().await?;
-            postgresql.start().await?;
+            postgresql.setup().await.unwrap();
```

Comment:
I'd at least `.expect("postgres did not install")` or something friendlier than a straight-up panic with a stack.